### PR TITLE
mesonlsp: init at 4.2.2

### PR DESCRIPTION
**Pull request by paveloom**
## Description of changes

https://github.com/JCWasmx86/Swift-MesonLSP

Depends on https://github.com/NixOS/nixpkgs/pull/269015.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by paveloom** [comment]
Added those as runtime dependencies.

**Comment by hcsch** [comment]
When trying to build this on top of your other PR for `swift-tools-support-core` I got a hash mismatch for the generated dependencies. Here's the diff I got after fixing those:

<details>
<summary>Diff from your generated files to the ones I got today (click to expand)</summary>

```patch
diff --git a/pkgs/by-name/sw/swift-mesonlsp/generated/default.nix b/pkgs/by-name/sw/swift-mesonlsp/generated/default.nix
index f6111300ff76..031954131be3 100644
--- a/pkgs/by-name/sw/swift-mesonlsp/generated/default.nix
+++ b/pkgs/by-name/sw/swift-mesonlsp/generated/default.nix
@@ -3,8 +3,8 @@
   workspaceStateFile = ./workspace-state.json;
   hashes = {
     "BitByteData" = "1bvbf89q6ivvy5q87k3jc07d5if5gvimz0shir8d6jw1pv5mn4ws";
-    "console-kit" = "0j249lzx9sywj80a9glwv480lzybf9a4cznhdfw5qrk7n7ckf13y";
-    "indexstore-db" = "02al5rk807685ian3xckpdl00yplm8v92b7gq1prw7abll8k6hnq";
+    "console-kit" = "1pn61rigw0svwig6d194sd0gqzfdxvra8vdkdyi5pqawra3xqgyd";
+    "indexstore-db" = "0plwcrh8z321x0i4psqg5lja2bm0lyc79winbx3v2jhdq5l1v9rg";
     "Perfect-INIParser" = "1v7q0bj4x87npanczn7yiicbg3c78wxjyww6cfkd1ipnvjh1gjmx";
     "sourcekit-lsp" = "1ybi92zspb20zx969xhcf9hzrjv5p01fjq9jibpk41pqqjch1whi";
     "SWCompression" = "1arjrx1w78v9p8ma8gp8fj79bggnp014433r46gxq8jja3qilwic";
@@ -15,12 +15,12 @@
     "swift-certificates" = "1kfgmwg9pdihnxq9b5r22b9w2fxcmciac24rl0az1v0zkzj495wz";
     "swift-collections" = "11d2vs2xxxbkvqqhm941y3an58qh52gfi6pv6xlar9873igijryq";
     "swift-crypto" = "1vng5mpbjx73w8z9a5j3mbpyq6mnqzc8rsqk9b3mckzznvvpih4m";
-    "swift-driver" = "108a1qr4syzngscb9lxil0r53l4ldjk56h0mr2jys52nlj543ad8";
-    "swift-llbuild" = "0q0ia2sfcdpkm29www34sdjgw9820xjbyc2gsi1gv6di0b5ar2aw";
+    "swift-driver" = "1cxl25kiz90kn4wldq64v9p1dyvggyfibf7f96anf130j9kg9rvx";
+    "swift-llbuild" = "19la95ld4vs3xlfvf560q1kq039f5dyj3rsmh0wzfaivxl9p5w83";
     "swift-log" = "17yshnwxr69d852a852l7dfhbja8gg7xah1pj7m5snah1q3k9vy7";
     "swift-nio" = "1gdhprgcp8kh9pg00j6rg7v3wcc7xkv5bgx2sfcdgcic1jh9lm0n";
-    "swift-package-manager" = "1wnyadisrwyjbb0xi3axpwrlxc9zjy0ckzg9j025c24saf1p952w";
-    "swift-syntax" = "1mz92x9hjs14wfp4qhj8dy35b7jkksj9lig6sm2rp2dqi6psbnxd";
+    "swift-package-manager" = "183pdf6m6rmh7ww854bzwvz47wkj9z4w4a3k697jasc8dka7lbmi";
+    "swift-syntax" = "18lvnz4j44f01srk129k2p4gjb143992ysqc2q3cwyfwsfk2gqcd";
     "swift-system" = "0402hkx2q2dv27gccnn8ma79ngvwiwzkhcv4zlcdldmy6cgi0px7";
     "swift-tools-support-core" = "0lmib1d16q0yidxaxl5szfcqwni9i1916pcqgy572h4i06qn20m2";
     "SwiftTreeSitter" = "13a8x2mms4dhv9rl94zv1jkmjbs718fawxnb0cw23qwnba17xv26";
diff --git a/pkgs/by-name/sw/swift-mesonlsp/generated/workspace-state.json b/pkgs/by-name/sw/swift-mesonlsp/generated/workspace-state.json
index d5658a76a613..91e112effb90 100644
--- a/pkgs/by-name/sw/swift-mesonlsp/generated/workspace-state.json
+++ b/pkgs/by-name/sw/swift-mesonlsp/generated/workspace-state.json
@@ -29,8 +29,8 @@
         },
         "state": {
           "checkoutState": {
-            "revision": "2e3e205e8d7563d5c1a6f1c8992616d337f632e6",
-            "version": "4.11.0"
+            "revision": "a7dd7001196d39b758e4990ec0f26f80162f4c84",
+            "version": "4.14.0"
           },
           "name": "sourceControlCheckout"
         },
@@ -47,7 +47,7 @@
         "state": {
           "checkoutState": {
             "branch": "main",
-            "revision": "89ec16c2ac1bb271614e734a2ee792224809eb20"
+            "revision": "5eac09c9a7b92303780e1ce7ea4aa5ee70226d56"
           },
           "name": "sourceControlCheckout"
         },
@@ -233,7 +233,7 @@
         "state": {
           "checkoutState": {
             "branch": "main",
-            "revision": "e0604b6195ccd53072c9455ee8f99b6bb094e911"
+            "revision": "d5c60fe47675e787f7589b04426e8aad127c90a8"
           },
           "name": "sourceControlCheckout"
         },
@@ -250,7 +250,7 @@
         "state": {
           "checkoutState": {
             "branch": "main",
-            "revision": "8333c9db6a0fde517c5527540b859d3ec41c5e08"
+            "revision": "bc3ffd54c2bfa8ce2214d1342860f07ae44922d1"
           },
           "name": "sourceControlCheckout"
         },
@@ -301,7 +301,7 @@
         "state": {
           "checkoutState": {
             "branch": "main",
-            "revision": "8e318dc5e3e7beafcd1f224daea10ffdc0ce8707"
+            "revision": "b368b96f9c3b5d360cd9e058fa106eff029c5fb3"
           },
           "name": "sourceControlCheckout"
         },
@@ -318,7 +318,7 @@
         "state": {
           "checkoutState": {
             "branch": "main",
-            "revision": "7c1c56c0aa505c939b210b9f3b6797417107820a"
+            "revision": "184930ad29dbfa924e9568f49daee8ad0f376223"
           },
           "name": "sourceControlCheckout"
         },
```
</details>

Not quite sure what would cause these changes since everything else seems to have stayed the same, especially the `Swift-MesonLSP` version.

**Comment by paveloom** [comment]
I can't reproduce this. I get the same hashes when running `swiftpm2nix` in the local copy of the [Swift-MesonLSP](https://github.com/JCWasmx86/Swift-MesonLSP) repository at the [v3.1.3](https://github.com/JCWasmx86/Swift-MesonLSP/releases/tag/v3.1.3) tag. To check the package build, I did the following:

```bash
git checkout swiftpm
git checkout -b swift-mesonlsp-merge
git cherry-pick swift-mesonlsp
nix build -Lv --rebuild .#swift-mesonlsp
```

**Comment by hcsch** [comment]
Sorry for only replying again now. I also just cherry-picked your swiftpm and swift-mesonlsp changes in my first attempt. To verify again I've now did exactly what you described, but this gets me the same error I got initially:
```
SWCompression> Submodule path 'Tests/Test Files': checked out '83a2bcadb7efb27c1835a2b1324b1a0617bcf901'
SWCompression> removing `.git'...
error: hash mismatch in fixed-output derivation '/nix/store/4d88l6lq8bvrhcks9v03g55ps7a68ac3-SWCompression-0bddafb.drv':
         specified: sha256-LHIa8VBSItyfIXkMQgK49r2VjnToPqQqummjw0PPMqs=
            got:    sha256-yz2f6OpbZv9fwiY0bd2SrImestoWtpss18sWZ42RbSU=
error: 1 dependencies of derivation '/nix/store/9iharryq2h7lzc65hafz3aiv7h0rghp1-swift-mesonlsp-3.1.3.drv' failed to build
```
I ran `nix build -Lv .#swift-mesonlsp`, as rebuild just got me an error about the swift-mesonlsp store path being an invalid output (which from a short search just seems to be the case if you haven't built the derivation before that you're trying to rebuild).

I'm also still confused as to what is happening here, since I should have exactly the same sources and inputs as you? Maybe some hashing doesn't work the same for you as it does for me? Maybe a fetcher doesn't produce the same results? I've tested this on both NixOS (running a flake-y system tracking nixos-unstable) and on Arch with a nix installed via pacman.

**Comment by hcsch** [comment]
I've also just now tried to get the same generated files with `swiftpm2nix` as you by making a fresh clone of Swift-MesonLSP and checking out `v3.1.3`, but I still get the diff I posted earlier.

To generate the files I ran `swift package resolve && swiftpm2nix` in the following dev-shell:
```nix
{
  inputs = {
    nixpkgs.url = "git+file:///path/to/nixpkgs/";
    flake-utils.url = "github:numtide/flake-utils";
  };
  outputs = { self, nixpkgs, flake-utils, }:
    flake-utils.lib.eachDefaultSystem (system:
      let
        pkgs = nixpkgs.legacyPackages.${system};
      in
      {
        devShells.default = pkgs.mkShell.override { stdenv = pkgs.swift.stdenv; } {
          buildInputs = with pkgs;[
            swift
            swiftpm
            swiftpm2nix
            swiftPackages.Foundation
          ];
        };
      });
}
```
The local nixpkgs clone is checked out on the swift-mesonlsp-merge branch created as described above.

Not sure if this is the recommended way to do this, but this is the first way that I tried that worked, after digging around some about how to get `swift package` to work at all (without complaining about there being no `Foundation` module).

**Comment by paveloom** [comment]
Okay, so

1) Swift's package manager caches packages and retrieves these packages from the cache when appropriate
2) Some of the [dependencies](https://github.com/JCWasmx86/Swift-MesonLSP/blob/v3.1.3/Package.swift) of Swift-MesonLSP are not pinned to a specific commit
3) `swift package resolve` doesn't retrieve updates to packages without a pinned commit

That's why I was getting the same hashes. I pruned Swift's package manager's cache with `swift package purge-cache`, and then got the new hashes with `swift package resolve && swiftpm2nix`. Note that I may have retrieved newer versions of some of the dependencies (e.g., hashes for `swift-driver` and `swift-package-manager` are different from yours).

Nixpkgs will cache Swift's packages after this pull request is merged, so this shouldn't be a problem. However, pinning the dependencies to a commit upstream would be beneficial.

**Comment by hcsch** [comment]
Thanks for clearing that up. I guess I should expect not quite guaranteed the same inputs made ok by caching a bit more.

**Comment by traxys** [comment]
Version 4.0.2 re-written in C++ was just recently released :/

File: pkgs/by-name/me/mesonlsp/package.nix
Comment (traxys) [inline pkgs/by-name/me/mesonlsp/package.nix:60]: Why the need to disable tests?

Comment (paveloom) [reply pkgs/by-name/me/mesonlsp/package.nix]: Well, because there was hardly any need for me to bother about it.

**Comment by traxys** [comment]
Result of `nixpkgs-review pr 269042` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mesonlsp</li>
  </ul>
</details>

**Review by traxys — APPROVED** [review]

**Review by traxys — APPROVED** [review]

File: pkgs/by-name/me/mesonlsp/package.nix
Comment (drupol) [inline pkgs/by-name/me/mesonlsp/package.nix:24]: I think we should use the `finalAttrs` pattern here.

The `finalAttrs` pattern will let you remove the `rec` keyword ([see its implementation in Nix](https://github.com/NixOS/nix/blob/2d1d81114d72ace89ce08cd3bc93f4eb27a2975d/src/libexpr/nixexpr.cc#L121)).

Why is this pattern preferred to `rec` ?

Let's take this simple code example:

```nix
mkDerivation rec {
   foo = 1;
   bar = foo + 1;
}
```

and then `.overrideAttrs(old: { foo = 2; })`, you'll get `{ foo = 2; bar = 2; }` while with `finalAttrs` pattern, it would work correctly because it's a real fixed point.

Let me share a couple of useful links regarding the `finalAttrs` pattern:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Feel free to reach out if you need some assistance.

File: pkgs/by-name/me/mesonlsp/package.nix
Comment (drupol) [inline pkgs/by-name/me/mesonlsp/package.nix:36]: Each patch in nixpkgs applied to the upstream source should be documented. Out-of-tree patches, fetched using `fetchpatch`, are preferred.

In each patch comment, please explain the purpose of the patch and link to the relevant upstream issue if possible. If the patch has been merged upstream but is not yet part of the released version, please note the version number or date in the comment such that a future maintainer updating the nix expression will know whether the patch has been incorporated upstream and can thus be removed from nixpkgs.

Furthermore, please use a _stable_ URL for the patch. Rather than, for example, linking to a GitHub pull request of the form `https://github.com/owner/repo/pull/pr_number.patch`, which would change every time a commit is added or the PR is force-pushed, link to a specific commit patch in the form `https://github.com/owner/repo/commit/sha.patch`.

Here are two good examples of patch comments:

```nix
mkDerivation {
  …

  patches = [
    # Ensure RStudio compiles against R 4.0.0.
    # Should be removed next 1.2.X RStudio update or possibly 1.3.X.
    (fetchpatch {
      url = "https://github.com/rstudio/rstudio/commit/3fb2397c2f208bb8ace0bbaf269481ccb96b5b20.patch";
      sha256 = "0qpgjy6aash0fc0xbns42cwpj3nsw49nkbzwyq8az01xwg81g0f3";
    })
  ];
}
```

```nix
mkDerivation {
  …

  patches = [
    # Allow building with bison 3.7
    # PR at https://github.com/GoldenCheetah/GoldenCheetah/pull/3590
    (fetchpatch {
      url = "https://github.com/GoldenCheetah/GoldenCheetah/commit/e1f42f8b3340eb4695ad73be764332e75b7bce90.patch";
      sha256 = "1h0y9vfji5jngqcpzxna5nnawxs77i1lrj44w8a72j0ah0sznivb";
    })
  ];
}
```

Comment (drupol) [reply pkgs/by-name/me/mesonlsp/package.nix:36]: Fair enough the filename is quite explicit.

**Review by drupol — CHANGES_REQUESTED** [review]

File: pkgs/by-name/me/mesonlsp/package.nix
Comment (drupol) [inline pkgs/by-name/me/mesonlsp/package.nix:141]: I would use `--replace-fail`

File: pkgs/by-name/me/mesonlsp/patches/disable-tests-that-require-network-access.patch
Comment (SuperSandro2000) [inline pkgs/by-name/me/mesonlsp/patches/disable-tests-that-require-network-access.patch:69]:
```suggestion
```
lets make the patch a bit shorter and easier to handle by just deleting the lines. Make sure to generate this locally, so that the line numbers above are correct

File: pkgs/by-name/me/mesonlsp/patches/disable-tests-that-require-network-access.patch
Comment (SuperSandro2000) [inline pkgs/by-name/me/mesonlsp/patches/disable-tests-that-require-network-access.patch:20]:
```suggestion
```

File: pkgs/by-name/me/mesonlsp/package.nix
Comment (SuperSandro2000) [inline pkgs/by-name/me/mesonlsp/package.nix:161]: We should probably just take svn, hg and git from path to not bloat the closure of the package with unnecessary things.

File: pkgs/by-name/me/mesonlsp/package.nix
Comment (SuperSandro2000) [inline pkgs/by-name/me/mesonlsp/package.nix:36]:
```suggestion
    ./disable-tests-that-require-network-access.patch
```

File: pkgs/by-name/me/mesonlsp/package.nix
Comment (drupol) [inline pkgs/by-name/me/mesonlsp/package.nix:1]: Can you please pass this new file through `nixfmt-rfc-style` ?

**Review by drupol — CHANGES_REQUESTED** [review]

**Review by drupol — APPROVED** [review]
LGTM

File: pkgs/by-name/me/mesonlsp/package.nix
Comment (GaetanLepage) [inline pkgs/by-name/me/mesonlsp/package.nix:139]:
```suggestion
  '';
  
  passthru = {
    updateScript = nix-update-script { };
    tests.version = testers.testVersion { package = mesonlsp; };
  };

  meta = with lib; {
```

**Review by drupol — APPROVED** [review]
LGTM

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 269042` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>mesonlsp</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 269042` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>mesonlsp</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 269042` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mesonlsp</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 269042` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mesonlsp</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Fails on darwin with:
```
Executing subproject muon:tinyjson

tinyjson| Project name: tiny-json
tinyjson| Project version: undefined
tinyjson| C compiler for the host machine: clang (clang 16.0.6 "clang version 16.0.6")
tinyjson| C linker for the host machine: clang ld64 609
tinyjson| Build targets in project: 8
tinyjson| Subproject tinyjson finished.

muon| Dependency tinyjson found: YES undefined (overridden)
muon| Dependency tracy skipped: feature tracy disabled
muon| Build targets in project: 9
muon| Subproject muon finished.

Run-time dependency nlohmann_json found: YES 3.11.3
Run-time dependency libcurl found: YES 8.8.0
Run-time dependency libarchive found: YES 3.7.4
Run-time dependency uuid found: NO (tried pkgconfig and framework)

meson.build:102:11: ERROR: Dependency "uuid" not found, tried pkgconfig and framework
```

**Comment by paveloom** [comment]
@GaetanLepage While I'm looking into this, I'm curious... do you just happen to have the four operating systems with these platforms? Or is there a way to cross-compile for Darwin while on Linux that I am missing?

**Comment by GaetanLepage** [comment]
Fails on darwin with:
```
Executing subproject muon:tinyjson

tinyjson| Project name: tiny-json
tinyjson| Project version: undefined
tinyjson| C compiler for the host machine: clang (clang 16.0.6 "clang version 16.0.6")
tinyjson| C linker for the host machine: clang ld64 609
tinyjson| Build targets in project: 8
tinyjson| Subproject tinyjson finished.

muon| Dependency tinyjson found: YES undefined (overridden)
muon| Dependency tracy skipped: feature tracy disabled
muon| Build targets in project: 9
muon| Subproject muon finished.

Run-time dependency nlohmann_json found: YES 3.11.3
Run-time dependency libcurl found: YES 8.8.0
Run-time dependency libarchive found: YES 3.7.4
Run-time dependency uuid found: NO (tried pkgconfig and framework)

meson.build:102:11: ERROR: Dependency "uuid" not found, tried pkgconfig and framework
```

> @GaetanLepage While I'm looking into this, I'm curious... do you just happen to have the four operating systems with these platforms? Or is there a way to cross-compile for Darwin while on Linux that I am missing?

Well, I only own `x86_64-linux` systems, but I have access to the ARM and darwin community builders.

**Comment by GaetanLepage** [comment]
On `aarch64-darwin`:
```
muon| Using subprojects/muon/subprojects/tinyjson.wrap

Executing subproject muon:tinyjson

tinyjson| Project name: tiny-json
tinyjson| Project version: undefined
tinyjson| C compiler for the host machine: clang (clang 16.0.6 "clang version 16.0.6")
tinyjson| C linker for the host machine: clang ld64 609
tinyjson| Build targets in project: 8
tinyjson| Subproject tinyjson finished.

muon| Dependency tinyjson found: YES undefined (overridden)
muon| Dependency tracy skipped: feature tracy disabled
muon| Build targets in project: 9
muon| Subproject muon finished.

Run-time dependency nlohmann_json found: YES 3.11.3
Run-time dependency libcurl found: YES 8.7.1
Run-time dependency libarchive found: YES 3.7.4
Run-time dependency uuid found: YES 1.6.2
Run-time dependency libpkgconf found: YES 2.2.0
Run-time dependency fmt found: NO (tried pkgconfig and framework)

meson.build:148:15: ERROR: Dependency "fmt" not found, tried pkgconfig and framework
```

**Comment by paveloom** [comment]
Stuck at figuring out why the [polyfill](https://github.com/JCWasmx86/mesonlsp/blob/v4.2.2/src/polyfill/polyfill.hpp) for the `std::format` function is not picked up on Darwin...

**Comment by traxys** [comment]
In the worst case the package could be marked linux only until someone with a darwin machine wants to debug it

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 269042` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 269042` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mesonlsp</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 269042` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mesonlsp</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 269042` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mesonlsp</li>
  </ul>
</details>

**Review by GaetanLepage — APPROVED** [review]
Looks like it builds fine on all platforms !

Good for me !

**Review by ghost — APPROVED** [review]
Been using this for a few days on `x86_64-linux` without issue.

**Pull request merged**
